<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>上传</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Z：</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>参数显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>参数显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>参数显示</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="palne1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="plane3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="plane2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>系统配置</string>
    </property>
    <addaction name="action"/>
    <addaction name="actionRTK_GPS"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>参数配置</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编队控制</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>队形变化</string>
     </property>
     <addaction name="action_mouse"/>
     <addaction name="actionkeyboard"/>
    </widget>
    <addaction name="menu_5"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu3D">
    <property name="title">
     <string>3D显示</string>
    </property>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>日志管理</string>
    </property>
    <addaction name="actionButton2"/>
    <addaction name="actionButton"/>
    <addaction name="action_13"/>
   </widget>
   <widget class="QMenu" name="menuQGC">
    <property name="title">
     <string>QGC</string>
    </property>
    <addaction name="actionqgc_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu3D"/>
   <addaction name="menu_4"/>
   <addaction name="menuQGC"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionarm"/>
   <addaction name="actiondisarm"/>
   <addaction name="separator"/>
   <addaction name="actiontakeOff"/>
   <addaction name="actionland"/>
   <addaction name="separator"/>
   <addaction name="actionauto"/>
   <addaction name="actionguided"/>
   <addaction name="actionposition"/>
   <addaction name="actionhome"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionConnect"/>
  </widget>
  <action name="actiontakeOff">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/1180624.png</normaloff>:/new/prefix1/image/1180624.png</iconset>
   </property>
   <property name="text">
    <string>takeOff</string>
   </property>
   <property name="toolTip">
    <string>起飞</string>
   </property>
  </action>
  <action name="actionland">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/1185255.png</normaloff>:/new/prefix1/image/1185255.png</iconset>
   </property>
   <property name="text">
    <string>land</string>
   </property>
   <property name="toolTip">
    <string>降落</string>
   </property>
  </action>
  <action name="actionposition">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/1154391.png</normaloff>:/new/prefix1/image/1154391.png</iconset>
   </property>
   <property name="text">
    <string>position</string>
   </property>
   <property name="toolTip">
    <string>定点</string>
   </property>
  </action>
  <action name="actionauto">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/1181722.png</normaloff>:/new/prefix1/image/1181722.png</iconset>
   </property>
   <property name="text">
    <string>auto</string>
   </property>
   <property name="toolTip">
    <string>自动</string>
   </property>
  </action>
  <action name="actionguided">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/534472.png</normaloff>:/new/prefix1/image/534472.png</iconset>
   </property>
   <property name="text">
    <string>guided</string>
   </property>
   <property name="toolTip">
    <string>跟随</string>
   </property>
  </action>
  <action name="actionarm">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/519987.png</normaloff>:/new/prefix1/image/519987.png</iconset>
   </property>
   <property name="text">
    <string>arm</string>
   </property>
   <property name="toolTip">
    <string>解锁</string>
   </property>
  </action>
  <action name="actiondisarm">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/519983.png</normaloff>:/new/prefix1/image/519983.png</iconset>
   </property>
   <property name="text">
    <string>disarm</string>
   </property>
   <property name="toolTip">
    <string>上锁</string>
   </property>
  </action>
  <action name="actionfd">
   <property name="text">
    <string>fd</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>固件选择</string>
   </property>
  </action>
  <action name="actionRTK_GPS">
   <property name="text">
    <string>RTK/GPS配置</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>光流设置</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>传感器设置</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>全部参数表</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>全部参数树</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>防碰撞模拟</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>智能算法</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>真实地形</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>虚拟地形</string>
   </property>
  </action>
  <action name="actionButton2">
   <property name="text">
    <string>下载日志</string>
   </property>
  </action>
  <action name="actionButton">
   <property name="text">
    <string>浏览日志</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>测试</string>
   </property>
  </action>
  <action name="action_mouse">
   <property name="text">
    <string> mouse</string>
   </property>
  </action>
  <action name="actionkeyboard">
   <property name="text">
    <string>keyboard</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/1175746.png</normaloff>:/new/prefix1/image/1175746.png</iconset>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
   <property name="toolTip">
    <string>连接</string>
   </property>
  </action>
  <action name="actionhome">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/1065350.png</normaloff>:/new/prefix1/image/1065350.png</iconset>
   </property>
   <property name="text">
    <string>home</string>
   </property>
   <property name="toolTip">
    <string>RTL</string>
   </property>
  </action>
  <action name="actionqgc_2">
   <property name="text">
    <string>qgc</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
